components/level/game.tsx

```diff
@@ -335,28 +335,28 @@ export default function Game({
         onNext();
       }
 
-      return;
+      return false;
```

Comment:
don't need to return a boolean here, or if we do it should be more clear what the boolean means